### PR TITLE
[Impreove][Master]remove no meaningful duplicated check code

### PR DESCRIPTION
**Pull request by Yao-MR-zz**
## Purpose of the pull request

remove no meaningful duplicated check code

## Brief change log

remove the duplicated check code

## Verify this pull request

This pull request is code cleanup without any test coverage.

This change added tests and can be verified as follows:

  - *remove the duplicated check code.*

### CODE REVIEW

**Comment by Yao-MR-zz** [comment]
just the dulicatede code

image

**Comment by yimaixinchen** [comment]
Well I don't think it is duplicate.if it removes, the below codes that the commands.size return NPE .

File: dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/MasterSchedulerService.java [hunk 1]
```diff
@@ -179,10 +179,6 @@ private void scheduleProcess() throws Exception {
     }
 
     private List<ProcessInstance> command2ProcessInstance(List<Command> commands) {
-        if (CollectionUtils.isEmpty(commands)) {
-            return null;
-        }
-
         List<ProcessInstance> processInstances = new ArrayList<>(commands.size());
```
Comment (yimaixinchen) [inline dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/MasterSchedulerService.java:182]: it returns NPE when commands is null

Comment (Yao-MR-zz) [reply dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/MasterSchedulerService.java:182]: @yimaixinchen 

first we know the function is for transform the command to process instance,
so we assue the command is not null, and also the we can know that the function who call the command2ProcessInstance have the predicate that the command is null or not null

image

**Comment by Yao-MR-zz** [comment]
hi @caishunfeng  can you help check this pr ?

**Review by caishunfeng — APPROVED** [review]
Thanks for your contribution, and I think this is an small change, looking forward to your next more constructive pr.

**Pull request merged**